app/services/record_declarations/npq_course_proxy.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module RecordDeclarations
+  class NPQCourseProxy
+    include NPQ
+  end
+end
```

Comment:
Why do we need this?

Reply:
The other option was to make the NPQ module callable directly.

Reply:
Or to embed the NPQCourseProxy inside the class that uses it. I'll do that.